pages/views/introduction/index.njk

```diff
@@ -6,4 +6,5 @@ order: -1
 tags: introduction
 ---
 
+<esl-d-rtl-switcher target="::next"></esl-d-rtl-switcher>
```

Comment:
Please, use exact target pages like tabs examples.
We can not have a temporary version with not ready or incorrectly placed content in the main branch.